src/styles/block/stars.css

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  width: 16px;
+  height: 16px;
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars__star:not(:last-child) {
```

Comment:
[checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) 4th item, you should have all the modifiers selectors, from 1 to 5